src/main/java/io/blamer/bot/conversation/routes/Registry.java

```diff
@@ -61,31 +63,36 @@ public class Registry implements Conversation {
   @Value("${answers.registry.description}")
   private String description;
 
+  /*
+  * @todo #73:45min\DEV Replace RSocket
+  *   We should replace rsocket with websocket
```

Comment:
@l3r8yJ why do we need websocket? Or I miss something..

Reply:
@h1alexbel It seems to me that rsocket is overcomplicated, or do you want to keep it?

Reply:
@l3r8yJ 
I don't understand the purpose of having websocket here

We can have simple http client here

Reply:
@h1alexbel i think for streaming `r/websockets` just native

Reply:
@l3r8yJ what kind of flow we discuss by the way?
**streaming is only needed in receiving new notification streams**, [here](https://github.com/Blamer-io/bot/blob/master/diagrams/notifications-read.svg) I described it pretty in details

any thoughts on this?

Reply:
@h1alexbel sure, let's switch to webclient for example, or we can use [feign clients](https://docs.spring.io/spring-cloud-openfeign/docs/current/reference/html/)